src/Generator.h

```diff
@@ -939,6 +945,30 @@ class GeneratorParam : public Internal::GeneratorParamImplBase<T> {
         : Internal::GeneratorParamImplBase<T>(name, value) {}
 };
 
+// Explicitly initiate std::string type outside to avoid constructor duplicate signature.
```

Comment:
This works but is dramatically different in style from how all of the other GeneratorParam subtypes are implemented; mimicking the existing style would arguably be better, though probably more verbose.

Reply:
The comment block above here (line 898 or so) is now out of date and needs updating.